src/System.Private.ServiceModel/tools/scripts/RemoveFirewallPorts.cmd

```diff
@@ -0,0 +1,39 @@
+@if "%_echo%" neq "on" echo off
+setlocal
+
+REM
+echo Start Remove all ports WCF test services use
+REM
+
+SET __EXITCODE=0
+
+netsh advfirewall firewall delete rule name="_WCF Test Server PortHttp"
```

Comment:
Nit: this could be made simpler by replacing :CheckError with a :DeleteRule that calls netsh and checks the error. It would eliminate duplication of strings and look like
```
call :DeleteRule "_WCF TestServer PortHttp"
call :DeleteRule "_WCF TestServer PortHttps"
etc
```